conn.go

```diff
@@ -322,11 +351,15 @@ func (conn *Conn) inWorker() {
 					Name:   iface + "." + member,
 					Body:   msg.Body,
 				}
-				conn.signalsLck.Lock()
-				for _, ch := range conn.signals {
-					ch <- signal
+				conn.signalQueueLck.Lock()
+				if conn.signalQueueWakeUp != nil {
```

Comment:
I set it to nil when closing the connection to avoid putting another signals in queue and thus sigWorker won't be woken up unnecessarily. Sounds sensible?